practice/src/css/bases/typography.css

```diff
@@ -0,0 +1,17 @@
+@font-face {
+  font-family: "Barlow";
```

Comment:
could you double check the name? 3 fonts are same

Reply:
I refer to this article [](https://www.smashingmagazine.com/2013/02/setting-weights-and-styles-at-font-face-declaration/ )and I have listed the advantages and disadvantages of the methods in Slack. So I choose Style Linking to give the font-family the same name. Do you agree with how I use it?

Reply:
Yes, I agree but are you missing `font-weight` and `font-style` ?